librosa/display.py

```diff
@@ -1098,30 +1106,21 @@ def __decorate_axis(
         axis.set_major_locator(MaxNLocator(prune=None, steps=[1, 1.5, 5, 6, 10]))
         axis.set_label_text("Time")
 
-    elif ax_type == "s":
-        axis.set_major_formatter(TimeFormatter(unit="s", lag=False))
+    elif isinstance(ax_type, str) and ax_type in time_units.keys():
```

Comment:
No need for `.keys()` here, can just say `ax_type in time_units`.

I'm also not sure about the type check here.  All the keys are strings anyway, so any ax_type which fails a string type check will also fail the membership test.

The only reason to do a type check here would be to short-circuit fail the membership test, which could be useful if `ax_type` is ever a non-hashable object.  Can that situation arise?